src/Menu/__snapshots__/Menu.component.spec.tsx.snap

```diff
@@ -10,7 +10,7 @@ exports[`Component: Menu should be defined 1`] = `
   display: flex;
   width: 100%;
   min-width: 250px;
-  background-color: primary.base;
+  background: primary.base;
```

Comment:
The `ThemeProvider` should probably be added to the Menu spec, but thats out of scope for this PR